### PR TITLE
Add a public road map

### PR DESCRIPTION
**Pull request by ianopolous**

### CODE REVIEW

File: RoadMap.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+![Peergos Logo](https://peergos.org/theme/img/peergos/logo-main.svg)
+
+Peergos Road Map
+========
+
+
+Centralized Alpha
+------------
+ - Stable web interface and data formats
+ - Streaming E2E encrypted video
```
Comment (cboddy) [inline RoadMap.md]: disabled by default

File: RoadMap.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+ - Streaming download of arbitrarily large files
```
Comment (cboddy) [inline RoadMap.md]: disabled by default

File: RoadMap.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+Decentralized writes
+------------
+ - Change MutablePointers to use authenticated pub-sub in IPFS
+ - Move to CRDTs for filesystem and btree
+
+Full decentralization
+------------
+ - Move core node's PKI data to append only log in IPFS
+ - Zk-SNARKs for follow requests?
```
Comment (cboddy) [inline RoadMap.md]: add link to zk-snarks paper(s)

File: RoadMap.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+ - Each user designates a Tor hidden service (publicly in the WriterData) to send follow requests to them to
+
+Keymail
+------------
+ - Application to display and edit text (ideally by granting app write access only to a hidden keymail folder)
```
Comment (cboddy) [inline RoadMap.md]: Not sure about this, what about if I want to use ``$EDITOR``? Would this  be better as a service (or  a service alongside an app?)

File: RoadMap.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+Group chat
+------------
+ - owner invite others
+ - owner can grant admin to others (admin => they can invite new users)
+ - allow inline pictures
+ - be careful about leaking social metadata accidentally
+
+Social Feed
+------------
+ - Essentially just a group chat that is for most (all?) your friends
```
Comment (cboddy) [inline RoadMap.md]: This is pretty vague.

File: RoadMap.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+
+Fully Quantum proof
+------------
+ - Move asymmetric crypto (follow requests and signing roots) to a post-quantum algorithm
```
Comment (cboddy) [inline RoadMap.md]: link to papers?

File: RoadMap.md (hunk 4 above)
Comment (ianopolous) [reply RoadMap.md]: Everything has to be usable through the web ui. We can also use it to build/test an application api. To use any text editor the editor would have to understand whatever format/protocol we agree on, and would have to go through the fuse api.

**Pull request merged**